### ISSUE TITLE
False Positive for ParameterListWrapping

### ISSUE THREAD

**Issue by adamu** [pinterest/ktlint#861]
Follow up from https://github.com/pinterest/ktlint/pull/786#issuecomment-680874240

## Expected Behavior
ParameterListWrapping is not raised.

## Observed Behavior
ParameterListWrapping is raised after upgrading to Detekt 1.12.0, but not before.

Fails with `Argument should be on a separate line (unless all arguments can fit a single line)` and `Missing newline before ")"` .

## Steps to Reproduce
```kotlin
foo(bar.apply {
    // stuff
})
```

## Your Environment

* Version of ktlint used: 0.38.1 (according to [detekt changelog](https://detekt.github.io/detekt/changelog.html))
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): `./gradlew detekt`
* Version of Gradle used (if applicable): 6.6.1
* Operating System and version: OSX 10.15.6
